server/sonar-server/src/test/java/org/sonar/server/platform/ws/ServerWsTest.java

```diff
@@ -29,7 +31,7 @@
 
 public class ServerWsTest {
 
-  WsTester tester = new WsTester(new ServerWs());
+  WsTester tester = new WsTester(new ServerWs(new ServerWsAction[]{new DummyServerWsAction()}));
```

Comment:
This is not testing Pico injection, this is testing code looping over injected actions in `WsTester`
(and was actually useful since this a way I found that had forgotten to write the loop at first)